### DISCUSSION TITLE
Collapsing multiple commits in the change log + Issue tracker integration

### DISCUSSION THREAD

**Discussion by aleksandr-seleznyov** [mooltiverse/nyx#272] [Q&A] [answered]
Hello!
First of all - thanks for a great product!
I have a question. 
Let's say I want my changelog to present only a single line for each task, even if there are multiple commits for this task. The task ID is shown in each commit message.
Is it possible currently? If yes - how?

Another question - can I somehow integrate with Jira or another issue tracker to pull task headlines for the changelog rather than relying on the developer to write proper messages?

**Comment by flelli** [comment, marked as answer]
Hi @aleksandr-seleznyov ,

I'm not sure I got your first question right but as far as I understand (also interpolating with the second question) Nyx has no concept like a _task_, only commits. I guess by _task_ you mean a Jira issue (or an issue from some other issue tracker) that you link by some means in the commit message, but again, there is no such concept in Nyx and, to your second question, there is no integration with Jira or other issue trackers.

Customizing the [template](https://mooltiverse.github.io/nyx/guide/user/configuration-reference/changelog/#template) is not enough for what you need.

I believe that the most Nyx can do for you is makeing the [changelog object model](https://mooltiverse.github.io/nyx/guide/user/state-reference/changelog/) available (as part of the [state](https://mooltiverse.github.io/nyx/guide/user/state-reference/) that can save on disk), but then you need to implement your own script to extract and roll up the information you need.

**Reply by aleksandr-seleznyov** [reply]
Thanks!
Yes, you got my questions correctly; thanks for the tips!
